src/ivan_cherednikov_ex/ex05/myrand.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Ivan Cherednikov'
+__email__ = 'ivch@nmbu.no'
+
+
+class LCGRand:
+
+    def __init__(self, seed):
+        self.seed = seed
+        self.a = 7 ** 5
+        self.m = 2 ** 31 - 1
```

Comment:
Make these class variable. You can check on the class skeleton from the requirement